pinot-core/src/main/java/org/apache/pinot/core/operator/query/SelectionOrderByOperator.java

```diff
@@ -117,6 +118,31 @@ public String toExplainString() {
     return stringBuilder.append(')').toString();
   }
 
+  private boolean isAllOrderByColumnsSorted() {
+    // Order by col1, col2 limit k will take all the n matching rows
```

Comment:
Let's revise the comments to match the functionality and put it as javadoc for this method